internal/pkgecosystem/download_test.go

```diff
@@ -0,0 +1,43 @@
+package pkgecosystem
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestDownloadNpmArchive(t *testing.T) {
+	tests := []struct {
+		name       string
+		pkgName    string
+		pkgVersion string
+		wantErr    bool
+	}{
+		{"download 'abc' with latest version", "abc", "", false},
```

Comment:
Is there a more legitimate package we can use other than "abc" ?

Reply:
Haha, sure